### PR TITLE
Add tutorial file to CubitPy

### PR DESCRIPTION
**Pull request by BishrMaradni**
Add a tutorial that aims to clarify and demonstrate the functionality of CubitPy.
This tutorial is based on the Cubit tutorial provided within Cubit 15.2

### CODE REVIEW

**Comment by isteinbrecher** [comment]
@BishrMaradni I added some comments / explanations / functionality. I suggest you have a final look and then we can merge this (I also included a small change in the package configuration). I did not have time to add testing for the tutorial, but that is something we can consider in the future. 

Thanks for the effort @BishrMaradni

File: cubitpy/cubitpy_types.py [hunk 1]
```diff
@@ -165,18 +165,15 @@ def get_baci_name(self):
...
-            return "SOLIDSH8"
+        if (
+            self == self.hex8
+            or self == self.hex20
+            or self == self.hex27
+            or self == self.tet4
+            or self == self.hex8sh
+            or self == self.tet10
+        ):
+            return "SOLID"
```
Comment (isteinbrecher) [inline cubitpy/cubitpy_types.py:176]: The old/new solid implementation is a little bit more complex as can be covered by this simple change. Let's merge the tutorial and talk about how to best implement this. For example, the `self.hex8sh` is a special solid shell element that is not yet converted by the new solid element implementation, also we might want to keep the option to create "old" solid elements until this issue is fully resolved in BACI.

Comment (BishrMaradni) [reply cubitpy/cubitpy_types.py]: Reverted as discussed

**Review by isteinbrecher — APPROVED** [review]

**Comment by isteinbrecher** [comment]
@BishrMaradni you can merge if you think this is ready.

**Pull request merged**